pkg/worker/image.go

```diff
@@ -134,10 +142,17 @@ func blobfsAvailable(path string) bool {
 }
 
 func (c *ImageClient) PullLazy(request *types.ContainerRequest) error {
-	imageId := request.ImageId
+	f, err := newLogger(request)
```

Comment:
Is the reason for the struct so that we can also log to stdout? Seems like we might be able to just have a function on the existing Logger similar to what you have below (`openLogFile`) which is like OpenContainerLogFile or something that just returns the file instead of a new struct

Reply:
Ok I went ahead and put it all in a method on the `ContainerLogger`